candi/openapi/doc-ru-init_configuration.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersions:
+- apiVersion: deckhouse.io/v1
+  openAPISpec:
+    description: |
+      Начальные параметры Deckhouse.
+
+      Параметры конфигурации с которой запустится Deckhouse после установки.
```

Comment:
I was confused wheh first read this, because here are two unrelated sentences.

I think, first sentence is enough.

Reply:
Paraphrased